UI/org.eclipse.birt.report.debug.core/src/org/eclipse/birt/report/debug/core/i18n/nls.properties

```diff
@@ -1,12 +1,16 @@
 #/*******************************************************************************
 # * Copyright (c) 2006, 2008 Actuate Corporation.
-# * All rights reserved. This program and the accompanying materials
-# * are made available under the terms of the Eclipse Public License v1.0
-# * which accompanies this distribution, and is available at
-# * http://www.eclipse.org/legal/epl-v10.html
-# *
+# * All rights reserved.
```

Comment:
This line is not needed. If we remove it, it will be in line with other projects.

Reply:
I resolved these two issues a couple of days ago.  Can you review?